From e49a0d4a8f3f725d6f683854e1cad36a3cd02962 Mon Sep 17 00:00:00 2001
From: Tony Tascioglu <tony.tascioglu@windriver.com>
Date: Wed, 19 May 2021 19:43:56 -0400
Subject: [PATCH] Remove fuzz testing from executing with ptests.

Upstream version 2.9.12 introduced new fuzz-testing and a corresponding
folder fuzz. These tests are not required for ptests of this package.

This patch removes the fuzz testing targets from the Makefile.
Otherwise, running the ptests will fail due to the invalid directory.

Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Tony Tascioglu <tony.tascioglu@windriver.com>
---
 Makefile.am | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index a9284b95..3d7b344d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,9 +2,9 @@

 ACLOCAL_AMFLAGS = -I m4

-SUBDIRS = include . doc example fuzz xstc $(PYTHON_SUBDIR)
+SUBDIRS = include . doc example xstc $(PYTHON_SUBDIR)

-DIST_SUBDIRS = include . doc example fuzz python xstc
+DIST_SUBDIRS = include . doc example python xstc

 AM_CPPFLAGS = -I$(top_builddir)/include -I$(srcdir)/include

@@ -210,7 +210,6 @@ runtests: runtest$(EXEEXT) testrecurse$(EXEEXT) testapi$(EXEEXT) \
 	    $(CHECKER) ./runxmlconf$(EXEEXT)
 	@(if [ "$(PYTHON_SUBDIR)" != "" ] ; then cd python ; \
 	    $(MAKE) tests ; fi)
-	@cd fuzz; $(MAKE) tests

 check: all runtests

--
2.25.1

